compose.testing.yml

```diff
@@ -2,3 +2,26 @@ services:
   backend-unit-tests:
     build: ./backend
     command: ["uv", "run", "pytest", "tests", "-s"]
+  backend:
```

Comment:
Added playwright service here, and included frontend and backend as dependencies.

A healthcheck is required otherwise playwright spins up and fails instantly since the frontend/backend take longer to startup.

Not sure what the convention is, this file will probably break if you run `docker compose up` but works fine if you run `playwright` or `backend-unit-tests` on their own